applications/ChimeraApplication/custom_strategies/strategies/fs_strategy_for_chimera.h

```diff
@@ -0,0 +1,762 @@
+            converged = BaseType::CheckFractionalStepConvergence(norm_dv);
+
+            if (converged)
+            {
+                KRATOS_INFO_IF("FSStrategyForChimera ", BaseType::GetEchoLevel() > 0 && Rank == 0)<<
+                    "Fractional velocity converged in " << it+1 << " iterations." << std::endl;
+                break;
+            }
+        }
+
+        // Activate Constraints for PRESSURE and deactivate VELOCITY
+        SetActiveStateOnConstraint(FS_CHIMERA_VELOCITY_CONSTRAINT, false);
+        SetActiveStateOnConstraint(FS_CHIMERA_PRESSURE_CONSTRAINT, true);
+
+        KRATOS_INFO_IF("FSStrategyForChimera ", (BaseType::GetEchoLevel() > 0 && Rank == 0) && !converged)<<
+            "Fractional velocity iterations did not converge "<< std::endl;
+
+        // Compute projections (for stabilization)
+        r_model_part.GetProcessInfo().SetValue(FRACTIONAL_STEP,4);
+        ComputeSplitOssProjections(r_model_part);
+
+        // 2. Pressure solution (store pressure variation in PRESSURE_OLD_IT)
+        r_model_part.GetProcessInfo().SetValue(FRACTIONAL_STEP,5);
+
+    #pragma omp parallel
+        {
+            ModelPart::NodeIterator nodes_begin;
+            ModelPart::NodeIterator nodes_end;
+            OpenMPUtils::PartitionedIterators(r_model_part.Nodes(),nodes_begin,nodes_end);
+
+            for (ModelPart::NodeIterator it_node = nodes_begin; it_node != nodes_end; ++it_node)
+            {
+                const double old_press = it_node->FastGetSolutionStepValue(PRESSURE);
+                it_node->FastGetSolutionStepValue(PRESSURE_OLD_IT) = -old_press;
+            }
+        }
+
+        KRATOS_INFO_IF("FSStrategyForChimera ", BaseType::GetEchoLevel() > 0 && Rank == 0)<<
+            "Calculating Pressure."<< std::endl;
+        //double norm_dp = 0;
+        double norm_dp = BaseType::mpPressureStrategy->Solve();
+
+#pragma omp parallel
+        {
+            ModelPart::NodeIterator nodes_begin;
+            ModelPart::NodeIterator nodes_end;
+            OpenMPUtils::PartitionedIterators(r_model_part.Nodes(),nodes_begin,nodes_end);
+
+            for (ModelPart::NodeIterator it_node = nodes_begin; it_node != nodes_end; ++it_node)
+                it_node->FastGetSolutionStepValue(PRESSURE_OLD_IT) += it_node->FastGetSolutionStepValue(PRESSURE);
+
+        }
+
+        // 3. Compute end-of-step velocity
+        KRATOS_INFO_IF("FSStrategyForChimera ", BaseType::GetEchoLevel() > 0 && Rank == 0)<<"Updating Velocity." << std::endl;
+        r_model_part.GetProcessInfo().SetValue(FRACTIONAL_STEP,6);
+        CalculateEndOfStepVelocity();
+
+        // Activate Constraints for PRESSURE and deactivate VELOCITY
+        SetActiveStateOnConstraint(FS_CHIMERA_VELOCITY_CONSTRAINT, true);
+        SetActiveStateOnConstraint(FS_CHIMERA_PRESSURE_CONSTRAINT, true);
+
+       // Additional steps
+        for (std::vector<Process::Pointer>::iterator iExtraSteps = BaseType::mExtraIterationSteps.begin();
```

Comment:
whats that?
I don't think we should use processes inside strategies

Reply:
No Idea, this is from the base fs_strategy. I guess it has something to do with turbulence models. 
https://github.com/KratosMultiphysics/Kratos/blob/3524897c3bffb920bde8d95f02be1917dce151e3/applications/FluidDynamicsApplication/custom_strategies/strategies/fs_strategy.h#L647